tonic/src/transport/channel/endpoint.rs

```diff
@@ -7,6 +7,7 @@ use crate::transport::service::TlsConnector;
 use crate::transport::Error;
 use bytes::Bytes;
 use http::uri::{InvalidUri, Uri};
+use http::HeaderValue;
```

Comment:
Can we combine these two imports since they come from the same crate?

Reply:
Missed this one. I'll push the change and merge.